packages/embeds/embed-core/embed-iframe-init.ts

```diff
@@ -17,6 +20,8 @@ export default function EmbedInitIframe() {
       ? embedNameSpaceFromQueryParam
       : window.name.includes("cal-embed=")
       ? window.name.replace(/cal-embed=(.*)/, "$1").trim()
+      : hasEmbedPath
```

Comment:
Gives ability to derive default namespace, even if embed query param is not present